cmd/runtime-manager/main.go

```diff
@@ -40,16 +43,22 @@ func main() {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 	pflag.Parse()
 
+	dir, _ := filepath.Split(options.RuntimeManagerEndpoint)
+	_ = os.Mkdir(dir, 0777)
```

Comment:
is it guaranteed to succeed?

Reply:
1. Handle error;
2. `os.MkdirAll` may be better;